### PR TITLE
feature(compiler): compilationMode partial

### PR DESCRIPTION
**Pull request by niwolf**
@plentymarkets/team-terra

#### Must be done by Terra

Testing

-   [ ] Person 1 (mandatory)

    Browser support (relevant for changes in scss / appearance of components)

    -   [ ] Chrome
    -   [ ] Firefox
    -   [ ] Safari

    Responsiveness

    -   [ ] Desktop
    -   [ ] Tablet
    -   [ ] Mobile

-   [ ] Person 2 (mandatory)

    Browser support

    -   [ ] Chrome
    -   [ ] Firefox
    -   [ ] Safari

    Responsiveness

    -   [ ] Desktop
    -   [ ] Tablet
    -   [ ] Mobile

Information transfer

-   [ ] Inform Ceres about changes in `TerraFormComponent` (optional: relevant for changes in this component)

Terra Basic Plugin

-   [ ] Adapt changes (optional: relevant for version updates, global design changes, l10n)

---

#### Must be done by every developer

Please inform a member of Terra (@plentymarkets/team-terra) to get the upper part of the checklist done (with urgency or deadline).

Documentation

-   [ ] Changelog (optional: relevant for every change which influences the API)
-   [ ] Example (updated or created)
-   [ ] JSDoc (optional: relevant for every change which influences the API)
-   [ ] [Google spreadsheet](https://docs.google.com/spreadsheets/d/1OINnux8TEoitV-qdAxqUQaf9oGI_fqFVwfRWenRFfhI/edit#gid=0) (relevant when deprecating public APIs or features)

Testing

-   [ ] Unit Test (updated or created)

### CODE REVIEW

File: src/lib/components/file-browser/terra-file-browser.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { EventEmitter } from '@angular/core';
+import { TerraStorageObject } from './model/terra-storage-object';
+import { TerraBaseStorageService } from './terra-base-storage.interface';
+
+/** Interface that acts as a lightweight injection token for the {@link TerraFileBrowserComponent}. */
+export abstract class TerraFileBrowser {
```
Comment (niwolf) [inline src/lib/components/file-browser/terra-file-browser.ts:6]: Die Schreibweise in Großbuchstaben für tokens fände ich auch nicht verkehrt.. Was meinst du?

Comment (pweyrich) [reply src/lib/components/file-browser/terra-file-browser.ts:6]: Habe mich doch dagegen entschieden. Ist ja nicht nur ein Token, sondern tatsächlich ne Klasse.

Comment (niwolf) [reply src/lib/components/file-browser/terra-file-browser.ts:6]: ok passt

**Comment by niwolf** [comment]
👍  kann nicht approven

**Comment by niwolf** [comment]
sieht gut aus & tested 👍

**Pull request closed without merging**